### DISCUSSION TITLE
What frequencies do you monitor?

### DISCUSSION THREAD

**Discussion by homeautomaton** [merbanan/rtl_433#2458] [Polls]
Do you have rtl_433 continually running with one or more devices, monitoring certain frequencies? The poll feature here only allows one answer, so make it your top choice, and comment if you continually monitor multiple frequencies, and whether you have more than one device to do it, or you use the "hop" feature.

It would also be interesting to know which frequencies are used in which countries. If you like, add a comment with your country, in addition to the info above.

Also, it takes two steps to vote. After you click a choice, **you must also click VOTE.**

**Comment by gdt** [comment]
I'm in the US, and monitor 915 for Ecowitt, 433 for acurite and 315 for TPMS, separate dongles.

**Comment by homeautomaton** [comment]
I'm in the US, and presently have 433 monitored 24x7. Working on adding 315 next, and eventually 868 for some motion sensors I have, but haven't played with.

**Comment by dimatx** [comment]
433 for Govee Water and Acurite refrigerator/freezer temperature. And 915 for Ecowitt WH51.

**Comment by hdtodd** [comment]
I'm in the US.  I continually monitor just 433.92MHz, having scanned some of the other frequencies and found no identifiable devices in my (rural) neighborhood.  I primarily monitor Acurite and LaCrosse thermometers (mine and my neighbors').

**Comment by keyboarderror** [comment]
I've listened occasionally over the years. Most traffic is on 433.92 MHz in my rural American neighborhood. Surveying other frequencies, 315 MHz shows some TPMS, and 345 MHz picks up a neighbor's home security system. Nothing heard so far on 868 and 915. The transformative discovery for me was the Home Assistant rtl_433 add on and the ability to easily create a database with mqtt, and act on it. I now have two receivers on 433.92 MHz to track various sensors and I'm trying to add a third to track some X10 hardware on 310 MHz. Currently they decode but they're not showing up in the database that I can tell.
